Gemfile.lock

```diff
@@ -230,35 +231,36 @@ GEM
     loofah (2.2.2)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    mail (2.6.6)
-      mime-types (>= 1.16, < 4)
-    mailcatcher (0.6.5)
-      eventmachine (= 1.0.9.1)
-      mail (~> 2.3)
-      rack (~> 1.5)
-      sinatra (~> 1.2)
-      skinny (~> 0.2.3)
-      sqlite3 (~> 1.3)
-      thin (~> 1.5.0)
-    meta_request (0.4.3)
+    mail (2.7.0)
+      mini_mime (>= 0.1.1)
+    mailcatcher (0.2.4)
+      eventmachine
+      haml
+      i18n
+      json
+      mail
+      sinatra
+      skinny (>= 0.1.2)
+      sqlite3-ruby
+      thin
+    meta_request (0.6.0)
       callsite (~> 0.0, >= 0.0.11)
       rack-contrib (>= 1.1, < 3)
-      railties (>= 3.0.0, < 5.2.0)
+      railties (>= 3.0.0, < 6)
     method_source (0.9.0)
-    midi-smtp-server (2.1.2)
-    mime-types (3.1)
+    midi-smtp-server (2.1.4)
+    mime-types (3.2.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2016.0521)
+    mime-types-data (3.2018.0812)
+    mini_mime (1.0.1)
     mini_portile2 (2.3.0)
-    minitest (5.10.3)
+    minitest (5.11.3)
     mono_logger (1.1.0)
-    multi_json (1.12.2)
+    multi_json (1.13.1)
     mysql2 (0.4.9)
```

Comment:
I'd like to see `mysql2` at `0.4.10` since it includes a buildfix for newer mysql versions.

What do you think about a more aggressive dependency update in the future?